AndroidSDKTests/src/test/java/com/leanplum/internal/RequestTest.java

```diff
@@ -323,4 +338,30 @@ public void testGetRequestsWithEncodedStringStoredRequests() {
     return requests;
   }
 
+  private static class ThreadWaiter implements Waiter {
+    Instant t1, t2, t3, t4;
```

Comment:
for better readability, it would be better to give these meaningful names:

beforeReadTime,
afterReadTime,
....